tests/integration/a2a/test_outbound_protocol.py

```
@@ -0,0 +1,197 @@
+    This is how every real outbound delegation gets its endpoint, so
+    if this test breaks, every cross-process A2A call breaks with it."""
+    from cuga.backend.cuga_graph.nodes.cuga_supervisor.a2a_protocol import _agent_card_rpc_url
+
+    class _Iface:
+        url = "http://peer.example:8002"
+        protocol_binding = "JSONRPC"
+        protocol_version = "0.3.0"
+
+    class _ProtoCard:
+        url = ""  # absent on protobuf
+        supported_interfaces = [_Iface()]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)